evm/p2p/les.py

```diff
@@ -232,7 +232,7 @@ def decode_payload(self, rlp_data: bytes) -> _DecodedMsgType:
         # This is just to make Proofs messages compatible with ProofsV2, so that LightChain
         # doesn't have to special-case them. Soon we should be able to drop support for LES/1
         # anyway, and then all this code will go away.
-        if len(decoded['proofs']) == 0:
+        if not decoded['proofs']:
```

Comment:
Ditto

Reply:
Not fixed as per the other comment chain (and this change done in many other places).